main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	fmt.Println("This is entrypoint for backplane-api")
+	mainRouter := mux.NewRouter()
+	mainRouter.NotFoundHandler = http.NotFoundHandler()
+}
```

Comment:
This function isn't really doing anything at the moment.
You can create a simple handler and add a unit test for that.

Reply:
Updated, thanks